compio-driver/src/poll/op.rs

```diff
@@ -18,6 +18,26 @@ use super::{sockaddr_storage, socklen_t, syscall, Decision, OpCode, RawFd};
 use crate::op::*;
 pub use crate::unix::op::*;
 
+impl<
+    D: std::marker::Send + Unpin + 'static,
+    F: (FnOnce(D) -> BufResult<usize, D>) + std::marker::Send + std::marker::Sync + Unpin + 'static,
```

Comment:
Separate Arg and Return type, i.e., `Asyncify<F, A, R>` and `F: (FnOnce(A) -> BufResult<usize, R>)`